game_event/tests.py

```diff
@@ -265,3 +296,53 @@ def test_fail_add_player_to_event_already_in_it(self, client, saved_game_event,
         assert response.status_code == 200
         with pytest.raises(IntegrityError):
             response = client.post(url, {'answer': 'yes'})
+
+
+@pytest.mark.django_db
+class TestWeatherApi:
+    def test_one_valid_response(self, client, saved_game_event, player, mock_urlopen):
+        mock_response = mock.Mock()
+        mock_response.status = 200
+        mock_response.read.return_value = bytes('{"weather": [{"icon": "04d"}],"main": {"temp": 13.19}}', 'utf-8')
+        mock_urlopen.return_value = mock_response
+
+        client.force_login(player.user)
```

Comment:
Is this line needed? Will it fail if we just remove it?

Reply:
(Same goes for the rest of the tests in this class)

Reply:
@yftacherzog Removed all.